src/app/screens/BookDetail/components/CommentsList/components/Comment/styles.ts

```diff
@@ -0,0 +1,30 @@
+import { StyleSheet } from 'react-native';
+import { CHARCOAL } from '@constants/colors';
+
+const styles = StyleSheet.create({
+  commentContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: 20
+  },
+  photo: {
+    borderRadius: 40,
+    height: 40,
+    marginTop: 5,
+    width: 40
+  },
+  description: {
+    display: 'flex',
+    flex: 1,
+    paddingLeft: 15
```

Comment:
Use `margin` to separate components, the `padding` is for spaces inside the component. Use `marginLeft: 15` or `marginRight: 15` in the photo styles